invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -96,40 +96,37 @@ <h2>{{ add_description.type.title_l10n }} <span
 
 
 {% macro _show_funding_item(item, index) %}
+  <dt class="ui tiny header">{{ item.funder.name if item.funder }}</dt>
   {%- if item.award -%}
-    {%- if item.award.title_l10n -%}
-      <dt class="ui tiny header">
-        <span class="mr-5">
-          {% if item.award.acronym %}
-            {{ item.award.acronym }} –
-          {% endif %}
-
-          {{ item.award.title_l10n }}
-        </span>
+  <dt class="ui tiny header">
+    <span class="mr-0">
+      {% if item.award.acronym %}
+        {{ item.award.acronym }} –
+      {% endif %}
+      
+      {%- if item.award.title_l10n -%}
+        {{ item.award.title_l10n }}
+      {%- endif -%}
+    </span>
 
-        {%- if item.award.number -%}
-          <span class="ui mini basic label ml-0 mr-5" id="number-label-{{ index }}">
-            {{ item.award.number }}
-          </span>
-        {%- endif -%}
+    {%- if item.award.number -%}
+      <span class="ui mini basic ml-0 text-muted {% if item.award.title_l10n %}mr-5{% endif %}" id="number-label-{{ index }}">
```

Comment:
And then maybe keep `label` to differentiate between the title and award number. I don't particularly love the box, but think there should be three styles in this section to differentiate the funder, title, and award number.